pkg/controller/plan/kubevirt.go

```diff
@@ -834,6 +834,56 @@ func (r *KubeVirt) EnsureGuestConversionPod(vm *plan.VMStatus, vmCr *VirtualMach
 	return
 }
 
+func (r *KubeVirt) EnsureVirtV2VPVCStatus() (ready bool, err error) {
+	pvcs := &core.PersistentVolumeClaimList{}
+	pvcLabels := map[string]string{
+		"migration": r.Migration.Name,
+		"ova":       "nfs-pvc",
+	}
+
+	err = r.Destination.Client.List(
+		context.TODO(),
+		pvcs,
+		&client.ListOptions{
+			LabelSelector: labels.SelectorFromSet(pvcLabels),
+			Namespace:     r.Plan.Spec.TargetNamespace,
+		},
+	)
+	if err != nil {
+		return false, err
+	}
+
+	if len(pvcs.Items) == 0 {
+		return
+	}
```

Comment:
when can it happen? what does it mean if we have 0 pvcs associated with the vm we import?

Reply:
right, missed that one - shouldn't we create the PVC in this case?

Reply:
maybe only for the first iteration when we sent the crate request for  the PVC but it didn't created yet

Reply:
this thread makes me think that we better call `EnsureVirtV2VPVCStatus` from `EnsureGuestConversionPod`..

Reply:
yeah, its sounds like a better solution and then we will query the pod status entirely rather than the PVC only. moved into `EnsureGuestConversionPod`.